internal/injector/aspect/advice/block.go

```diff
@@ -50,20 +68,44 @@ func (a *prependStatements) Apply(ctx context.AdviceContext) (bool, error) {
 }
 
 func (a *prependStatements) Hash(h *fingerprint.Hasher) error {
-	return h.Named("prepend-statements", a.Template)
+	return h.Named("prepend-statements", a.Template, fingerprint.Int(a.order), fingerprint.String(a.namespace))
 }
 
 func (a *prependStatements) AddedImports() []string {
 	return a.Template.AddedImports()
 }
 
+func (a *prependStatements) Order() int {
+	if a.order == 0 {
+		return DefaultOrder
+	}
```

Comment:
No biggie -- found this a little odd, considering `DefaultOrder` is also `0`. Maybe the default should be set at creation time?